0_RootFS/Go/build_tarballs.jl

```diff
@@ -3,22 +3,24 @@ using BinaryBuilder
 include("../common.jl")
 
 name = "Go"
-version = v"1.13"
+version = v"1.16.3"
 
+# https://golang.org/dl/
 sources = [
-    "https://dl.google.com/go/go1.13.linux-amd64.tar.gz" =>
-    "68a2297eb099d1a76097905a2ce334e3155004ec08cdea85f24527be3c48e856",
+    ArchiveSource(
+        "https://golang.org/dl/go1.16.3.linux-amd64.tar.gz",
+        "951a3c7c6ce4e56ad883f97d9db74d3d6d80d5fec77455c6ada6c1f7ac4776d2",
+    )
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 mv ${WORKSPACE}/srcdir/go ${prefix}/
 """
 
-# We only build for Linux x86_64
+# We only build for host platform: x86_64-linux-musl
 platforms = [
-    # TODO: Switch to musl once https://github.com/rust-lang/rustup.rs/pull/1882 is released
```

Comment:
TODO was copy and pasted from the Rust `build_tarballs.jl` file: https://github.com/JuliaPackaging/Yggdrasil/commit/2c40e825c#diff-9d82d1671f46cc0048c4743f450921e16dad3dfc855959d284a5513e25f2d531R9